src/test/java/software/aws/mcs/auth/TestSigV4Config.java

```diff
@@ -55,7 +55,7 @@ public static void main(String[] args) throws Exception {
         //By default the reference.conf is loaded by the driver which contains all defaults.
         //You can override this by providing reference.conf on the classpath
         //to isolate test you can load conf with a custom name
-        URL url = TestSigV4Config.class.getClassLoader().getResource("keyspaces-reference.conf");
+        URL url = TestSigV4Config.class.getClassLoader().getResource("keyspaces-reference-norole.conf");
```

Comment:
I would create a new test class for this feature. This looks like maybe you checked in some test work in progress

Reply:
Created a new class to test Assume IAM role config.